### DISCUSSION TITLE
Do you need devices to test with?

### DISCUSSION THREAD

**Discussion by code-in-progress** [sbidy/wiz_light#42] [Q&A]
Fantastic work mate. 

In the readme, it said you are looking for testing contributions? I have (or can get) some spare devices for you if you need them or I can do a PR and test whatever you want here against my setup. Whatever works for you to keep the development going on this. I just recently ditched nearly all of my Zigbee lights and switched them over to Wiz lights (36 bulbs so far and 14 more to go), so I REALLY want this integration to continue. :+1:

**Comment by sbidy** [comment]
Hey @code-in-progress - thank you for the feedback!
You can help to made the integration better if you test the new releases as soon they will release.
There are so many different types of Wiz Bulb out there - If test on a wide variety we push the overall quality.
And this also avoids the "works for me" mentality 😉 

In this case hit the "Watch" button in GitHub to be up to date if I made some changes to the code.
The **master** branch will always contain a working and formally tested version.
In the future there will be also a change log and "correct" versioning introduced.

**Comment by code-in-progress** [comment]
I completely forgot to add these to the tested list:

A21 Color Bulb (https://www.homedepot.com/p/Philips-Color-and-Tunable-White-A21-LED-100-Watt-Equivalent-Dimmable-Smart-Wi-Fi-Wiz-Connected-Wireless-LED-Light-Bulb-562405/313643239)
{"method":"getSystemConfig","env":"pro","result":{"mac":"a8bb504cb454","homeId":1445363,"roomId":2349572,"moduleName":"ESP03_SHRGB1W_01","fwVersion":"1.20.0","groupId":0,"drvConf":[20,2],"ewf":[200,255,255,255,0,0,40],"ewfHex":"c8ffffff000028"}}

This is the A19 Amber bulb (https://www.homedepot.com/p/Philips-Soft-White-A19-LED-40W-Equivalent-Dimmable-Smart-Wi-Fi-Wiz-Connected-Wireless-Light-Bulb-555516/310289035):
{"method":"getSystemConfig","env":"pro","result":{"mac":"a8bb509dd4f5","homeId":1445363,"roomId":2138309,"homeLock":false,"pairingLock":false,"typeId":0,"moduleName":"ESP06_SHDW9_01","fwVersion":"1.19.3","groupId":0,"drvConf":[20,1]}}

**Comment by code-in-progress** [comment]
> There are so many different types of Wiz Bulb out there - If test on a wide variety we push the overall quality.
> And this also avoids the "works for me" mentality wink

Agreed. I can also help on the coding side as well if you need it.

**Comment by sbidy** [comment]
> 
> 
> > There are so many different types of Wiz Bulb out there - If test on a wide variety we push the overall quality.
> > And this also avoids the "works for me" mentality wink
> 
> Agreed. I can also help on the coding side as well if you need it.

You welcome to fork and made pulls 😄  -- currently I've some trouble to get the config flow up and running ... see "ConfigFlow" branch.

**Comment by code-in-progress** [comment]
I'll take a look when I get off work tonight. I can pop over into the Discord later and see if I can grab a HA dev to give some advice on ConfigFlow. I've not written anything with it yet, but that's never stopped me before. LOL

**Comment by steveothesane** [comment]
@sbidy Here are two styles of the Philips Wiz bulbs I have been using with Home Assistant.  I have 5 of the color style and 5 of the Cold to Warm style.  I have had no problems with them at all using your add-on.  I included the screen shots from the Wiz app as it is the only way I know of to get the info such as, firmware version, you may need.  I'm sure there is another way to access them but i haven't figured it out without tearing them apart.  Have you found out how to give them static IP addresses or make any changes to the firmware?
![Phillips Wiz A19 Cold to Warm](https://user-images.githubusercontent.com/32602457/96328839-e948c380-0ffb-11eb-8883-6e47fc6662fb.png)
![Phillips Wiz A19 Color](https://user-images.githubusercontent.com/32602457/96328840-e9e15a00-0ffb-11eb-8650-3ea2738cc4b3.png)

**Comment by sbidy** [comment]
No, I'm not sure but this can only be done from the app/cloud.
In my opinion there must some kind of connection/api byside of the UDP.

**Comment by pana800** [comment]
How should I add new tested lamp to the list?
Successfully added LED Strip Model B27082 to HA. 
Color wheel, on off, CCT and effects are working.
Thanks for the great work

**Comment by sbidy** [comment]
Please create a new Issue in this repo. Please use the Issue Template "Bulb Test Report".
Thank you for the support.
